documentation/meta/incidents/runbooks/incident_response_runbook.md

```diff
@@ -0,0 +1,141 @@
+to use your best judgement here, but here are some examples to help you:
+
+- **Severity 1** examples:
+  - <https://openverse.org> is completely down.
+  - Frontend searches aren't returning any results
+  - The API is serving error codes for all searches
+- **Severity 2** examples:
+  - Frontend keyboard navigation is broken
+  - A particular provider with ~1.5m images is not returning any API results
+  - Users based in the UK, comprising 6% of all visitors, are unable to view
+    <https://openverse.org>
+- **Severity 3** examples:
+  - A piece of our staging infrastructure has completely failed.
+  - A staging site is down
+  - Greek frontend translations are not displaying, instead falling back to
+    English
+
+### Concluding Reporting
+
+Monitor the incident until a lead has been identified. This will be the MSR, by
+default or another online member of the team (perhaps even yourself). At this
+point, reporting the incident is complete.
+
+## 2. Stabilize service
+
+1. Complete any necessary communication work. For Severity 1, this means
+   messaging the maintainers via Signal that an outage is occurring. Depending
+   on the severity, a Make post announcing that the outage is being investigated
+   may also be warranted.
+   1. If the incident was discovered by a user in a communication channel,
+      please reply to them, and thank them for their report. Let them know the
+      team is investigating the issue.
+2. The point person should always have an up-to-date understanding of the
+   situation. Post updates to P2 regularly depending on severity (ideally hourly
+   for severity 1, but daily or otherwise for severity >= 2). Anyone who signs
+   on for the day while the incident is being triaged should jump into
+   stabilizing the service with the team. Be proactive! Ask how you can help.
+3. Perform any necessary tasks to triage/mitigate the incident. Delegate what
+   you can to online developers. It is assumed that anyone online can and will
+   help, and that incidents take precedence over regular work. Some tips for
+   triaging:
+   1. It’s possible we have an existing runbook related to the type of incident
+      you’re dealing with, particularly if it was triggered by a monitoring
+      alarm. Search the
+      [runbook documentation](/meta/monitoring/runbooks/index.md) for useful
+      information.
+      1. Understanding [service deployment mechanisms](/general/deployment.md)
+         will likely be useful.
+      2. Traffic anomalies? See
+         [this guide](/meta/monitoring/traffic/runbooks/identifying-and-blocking-traffic-anomalies.md)
+         for suggestions.
+   2. Be prepared to check logs in AWS, observe traffic in Cloudflare, and look
+      at dashboards for information on server resources. A useful guide on logs
+      [can be found here](/meta/monitoring/cloudwatch_logs/index.md).
+   3. Checking for differences between staging and production can also be
+      illuminating.
```

Comment:
This could use further explanation in a follow-up ticket, it's a bit vague and I wouldn't know where to start with this, necessarily, though I have some guesses based in my extensive knowledge of our infrastructure (which not every maintainer has nor should be expected to have).

Reply:
Issue made: https://github.com/WordPress/openverse/issues/3580